test/clamav.bats

```diff
@@ -0,0 +1,79 @@
+load 'test_helper/common'
+
+TEST_NAME_PREFIX='ClamAV:'
+CONTAINER_NAME='dms-test-clamav'
+RUN_COMMAND=('run' 'docker' 'exec' "${CONTAINER_NAME}")
+
+function setup_file() {
+  local PRIVATE_CONFIG
+  PRIVATE_CONFIG=$(duplicate_config_for_container . "${CONTAINER_NAME}")
+
+  docker run --rm --detach --tty \
+    --name "${CONTAINER_NAME}" \
+    --hostname mail.my-domain.com \
+    --volume "${PRIVATE_CONFIG}:/tmp/docker-mailserver" \
+    --volume "${PWD}/test/test-files:/tmp/docker-mailserver-test:ro" \
+    --env ENABLE_AMAVIS=1 \
+    --env AMAVIS_LOGLEVEL=2 \
+    --env ENABLE_CLAMAV=1 \
+    --env ENABLE_UPDATE_CHECK=0 \
+    --env ENABLE_SPAMASSASSIN=0 \
+    --env ENABLE_FAIL2BAN=0 \
+    --env PERMIT_DOCKER=host \
```

Comment:
You recently added the `PERMIT_DOCKER=host` line here, what was it needed for?

Reply:
The container was refusing the mails sent via `nc` with `DATA without valid RCPT from ...`, so I tried and tried, and found this toggle to solve the problem. `tests.bats` uses it the same way, presumably because `nc` does not respect proper protocol? There was definitely (improper) command pipelining involved, which seemed to cause issues too.. or at least warnings. I am not 100% sure why this solved the issue though..

Reply:
Ah ok, I am familiar with that issue, but assumed since it was run in the container that it shouldn't matter :sweat_smile: 

I assume it's because that without a `PERMIT_DOCKER` value, we get `none` which clears out the Postfix `main.cf:mynetworks` setting:

https://github.com/docker-mailserver/docker-mailserver/blob/70ad765a30cd5ce8385994f592f51e05b1e0a744/target/scripts/startup/setup-stack.sh#L663-L667

All good then, thanks for clarifying :grinning:

Reply:
You're welcome :)